src/surfaces/actionBar/actionBarSearchSelect.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Select from '../../inputs/select';
+import Divider from '../../dataDisplay/divider';
+import {
+    UI_CLASS_NAME,
+} from '../../global/constants';
+
+type SelectOption<TValue = number, TLabel = string> = {
+    label: TLabel;
+    value: TValue;
+};
+
+type PropTypes = {
+    dropdownArrowIconType?: string;
+    onChange: (value: SelectOption | number | string | null) => void;
+    options: SelectOption[];
+    placeholder?: string;
+    value: number | string | SelectOption | null;
+};
+
+const defaultProps: Pick<PropTypes, 'dropdownArrowIconType' | 'placeholder'> = {
+    dropdownArrowIconType: 'caret-down',
+    placeholder: 'Select',
+};
+
+function ActionBarSearchSelect(props: PropTypes) {
+    const {
+        dropdownArrowIconType,
+        onChange,
+        options,
+        placeholder,
+        value,
+    } = props;
+
+    return (
+        <React.Fragment>
+            <Select
+                // @ts-ignore
```

Comment:
Can we add another comment describing the TS issue we have to ignore?

Reply:
Absolutely, Added comment showing TS issue.

Reply:
Hmm, I think that's telling us we need add `className` to the prop type declarations for `<Select>`! =)
Can we try that and see if that makes the TS error go away?

Reply:
Nevermind, it's already there.  I think it's confused due to `forwardRef()`.
We'd have to either convert the `<Select>` component to TS, or add `select.d.ts` types file.
Let's leave this as-is for now! =)